spec/aep-64/README.md

```diff
@@ -0,0 +1,76 @@
+---
+aep: 64
+title: "JWT authentication for provider API"
+author: Artur Troian (@troian)
+status: Draft
+type: Standard
+category: Core
+created: 2025-04-03
+updated: 2025-04-03
+roadmap: major
+---
+
+## Motivation
+
+Enhance tenant authentication to the provider gateway using JWT allows client API to retain communication with leases during blockchain maintenance.
+
+## Summary
+
+mTLS authentication is great security mechanism, however, with current implementation when blockchain acts as root of trust, it becomes complicated for API clients to retain communication with leases during blockchain maintenance windows. There are also other limitations like granual access. Certificate gives access to all all leases and all features, tho granual access is possible via certificate, implementing and maintaining it for clients is challenging topic. JWT on other hand is wide adopted approach for authentication and authorization.
+
+## Technical details
+
+Akash uses asymetric keys (ECDSA) for wallets with secp256k1 curve and allows for signing and encrypting messages.
+For this AEP, we will be focused on signing part only.
+
+Conventionally, JWT is being issued by the server to the client upon authentication, with Akash Network, however, client will be issuing the JWT.
+Reasons for this are following:
+- single wallet may have multiple simultaneous leases on different providers
+- lease owner is only who can create granual access JWT, for example one that can only pull logs for certain lease(s) and/or service(s)
+- public key of the wallet is on the blockchain which allows any provider to validate authenticity of the signature
```

Comment:
Would be worth stating here that this public key is already on the blockchain and doesn't need publishing because we will be using the account's public key (available on-chain after the first transaction made with that account). Do you believe its relevant and accurate?

Reply:
Line 30 does state this?